src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchRestTemplate.java

```diff
@@ -127,12 +127,11 @@ public String index(IndexQuery query, IndexCoordinates index) {
 	}
 
 	@Override
-	@Nullable
-	public <T> T get(GetQuery query, Class<T> clazz, IndexCoordinates index) {
-		GetRequest request = requestFactory.getRequest(query, index);
+	public <T> Optional<T> get(String id, Class<T> clazz, IndexCoordinates index) {
```

Comment:
Let's stick with `T` instead of `Optional<T>`. A huge number of folks already asked us to roll back the `Optional` change at `findById` and `Optional` isn't being loved that much.